lib/explorer/backend/lazy_series.ex

```diff
@@ -59,6 +66,38 @@ defmodule Explorer.Backend.LazySeries do
     end
   end
 
+  for op <- @arithmetic_operations do
+    @impl true
+    def unquote(op)(%Series{} = left, value_or_series) do
```

Comment:
Should we be enforcing somehow that those are series with the LazySeries as backend? Maybe have a helper called `lazy_series!(series)`, that receives a series, checks the argument is a series with lazy backend, and return the data?

Reply:
Good point! I'm going to change that in a following PR.